### PR TITLE
doc update

### PR DESCRIPTION
**Pull request by jaxxzer**
Working on removing the few remaining todos.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -47,22 +45,6 @@ To begin using the device, connect it to a USB - UART converter that operates on
 
 Visit the [Device Manager](device-manager.md) menu for more connection options.
 
-## Principle of Operation
```
Comment (patrickelectric) [inline docs/index.md]: We are moving this specific information to somewhere else ?

File: docs/index.md [hunk 2]
```diff
@@ -96,26 +78,20 @@ There is an orange arrow on the Distance Axis indicating the distance to the tar
...
 
 The color scheme (gradient) of the Waterfall can be [customized](display-settings.md).
 
 ## Application and Device Settings
 
 See [here](display-settings.md) for instructions to switch between meters and feet for the application display.
 
-To adjust the update rate and other Ping device settings, visit the [device configuration](device-settings-ping1d) page.
-
-## Interpreting Profile Data
```
Comment (patrickelectric) [inline docs/index.md]: I believe that such section is not only for ping1d

File: docs/logging-and-user-files.md [hunk 3]
```diff
@@ -8,8 +8,6 @@ This section describes the user-accessible files used by the Ping Viewer applica
 
 Ping Viewer will create a folder called `PingViewer` in your user documents folder:
 
-TODO use full path
```
Comment (patrickelectric) [inline docs/logging-and-user-files.md]: Why are we removing this ? We are not using full, but relative paths.

File: docs/firmware-update.md [hunk 4]
```diff
@@ -48,5 +48,3 @@ If the led is not blinking:
      - Where /dev/ttyUSB0 is my serial port
 
 If the instructions are not clear or sound a bit complicated, please wait a bit more for us to create a Ping-Viewer version that allows you to recover devices.
-
-> TODO explain bootloader process
```
Comment (patrickelectric) [inline docs/firmware-update.md]: An issue should be created to the product documentation for that.

File: docs/index.md (hunk 1 above)
Comment (Williangalvani) [reply docs/index.md]: it is now at https://bluerobotics.com/learn/ping-sonar-technical-guide/#operating-principle

File: docs/logging-and-user-files.md (hunk 3 above)
Comment (Williangalvani) [reply docs/logging-and-user-files.md]: Is there a good reason to use full paths here? I don't know about macOs, but with Windows and Linux it is actually harder to get to these directories using the full path.

File: docs/firmware-update.md (hunk 4 above)
Comment (jaxxzer) [reply docs/firmware-update.md]: https://github.com/bluerobotics/ping-viewer/issues/764

File: docs/logging-and-user-files.md (hunk 3 above)
Comment (jaxxzer) [reply docs/logging-and-user-files.md]: > Is there a good reason to use full paths here?

no the todo is moot

File: docs/index.md (hunk 2 above)
Comment (jaxxzer) [reply docs/index.md]: > To adjust the update rate and other Ping device settings, visit the relevant page in the 'Device Settings' section.

Comment (jaxxzer) [reply docs/index.md]: It is beyond the scope of this documentation.

**Pull request merged**